doc/rst/language/spec/types.rst

```diff
@@ -634,9 +634,9 @@ concrete or semi-concrete
              writeln("Every reform movement has a lunatic fringe.");
           when statesman.Churchill do
              writeln("A joke is a very serious thing.");
-          when statesman.Kissinger do
-             { write("No one will ever win the battle of the sexes; ");
-               writeln("there's too much fraternizing with the enemy."); }
+          when statesman.Plato do
+             { write("I only wish that wisdom were the kind of thing that flowed...");
+               writeln("... from the vessel that was full to the one that was empty."); }
```

Comment:
Thank you for your patch! I think you might need to update the expected output of the test, as well. See like 657 specifically.

Reply:
Ah, I had totally missed that!  I've gone ahead and updated it.

Reply:
Additionally, I was double checking the attribution of the quote, and it looks like it ought to be attributed to Socrates? This [Wikiquote page](https://en.wikiquote.org/wiki/Socrates) suggests that Plato wrote down the saying in the PR's diff as having been said by Socrates.

Reply:
Ah, damn!  I can't quite remember the source (I found a good site resource for it but the attribution format was a little strange), but it looks like you're right.  Lemme try and find another one since Socrates is already in there.

Reply:
Oh, wait; it's Aristotle that's already in here, not Socrates.  Should I just update it to Socrates, then?

Reply:
Changed!  Also, excellent catch!  Thank you for double checking!

Reply:
Thank you! Using Socrates seems good to me. I am going on break for the holidays, but your PR will be near the top of my queue in the new year.

Reply:
Ah!  No no, take your time!  It's a small change, obviously.  You should be enjoying your holiday, not participating in PRs! :joy: 

Thank you either way, though!